grype-server/pkg/scanner/scanner.go

```diff
@@ -175,6 +175,11 @@ func (s *Scanner) loadBb(cfg *db.Config) error {
 	if err != nil {
 		return fmt.Errorf("failed to create curator: %v", err)
 	}
+	// Check IsUpdateAvailable() errors because if it has errors, the update process will be continued during Update().
+	// https://github.com/anchore/grype/blob/731abaab723ae8918635d4e20399ca3c00b665f4/grype/db/curator.go#L138-L143
+	if _, _, err := dbCurator.IsUpdateAvailable(); err != nil {
```

Comment:
let's try to corrupt the DB on purpose to simulate a crashed OOM flow, and then we can start the grype server locally to see if it goes through this flow, and whether we can recover from a corrupted DB situation somehow.

Reply:
ack